src/ext/logging.h

```diff
@@ -4,19 +4,21 @@
 
 #include "configuration.h"
 
-#if defined(ZTS) && PHP_VERSION_ID < 70000
-#define TSRMLS_FC TSRMLS_D,
-#define ddtrace_log_errf(...) _ddtrace_log_errf(TSRMLS_C, __VA_ARGS__)
-#else
-#define TSRMLS_FC
 #define ddtrace_log_errf(...) _ddtrace_log_errf(__VA_ARGS__)
```

Comment:
Why do we have this macro that just forwards to one of our own functions?

Reply:
It made sense when we had separate API's under the hood for each PHP version, but now that we've refactored it quite a bit, we can certainly get rid of it. :)